src/external/pcglAuthZClient.ts

```diff
@@ -61,13 +61,17 @@ export const fetchUserData = async (token: string): Promise<PCGLUserSessionResul
 
 	if (!response.ok) {
 		const errorResponse: UserDataResponseErrorType = await response.json();
-
 		logger.error(`Error retrieving user data.`, errorResponse);
+
 		switch (response.status) {
 			case 403:
-				throw new lyricProvider.utils.errors.Forbidden(errorResponse.error);
+				throw new lyricProvider.utils.errors.Forbidden(
+					'error' in errorResponse ? errorResponse.error : errorResponse.detail,
```

Comment:
Related to the type of this error... why not just have a standard `message` property that both halves of the union share?